src/outputProcessor/logsTxtFormatter.js

```diff
@@ -1,16 +1,17 @@
 const CONSTANTS = require("../constants");
 
 const PADDING = '    ';
-const PADDING_LOGS = `${PADDING}`.repeat(6);
+const PADDING_LOGS = `${PADDING}`.repeat(10);
```

Comment:
Changing this would require updating a lot of tests. I would suggest just leaving the padding the same and adding the time string above the log line. That way you can skip doing tests as well.

Reply:
Yes that's what I saw ;-) , plus I couldn't get it right anyway. And your idea is better, this way it's more similar to the terminal output. 
Sorry it took a few tries, I couldn't get the tests to pass locally, I think because of my corporate proxy.
Now the tests pass and I tried it on my project, the output looks ok to me:

```
                        Time: 1708517910207
        cy:request (K): PATCH http://localhost:XXXX/emulator/stop
                        Status: 200
                        Response body: <EMPTY>
                        Time: 1708517910210
        cy:command (K): cypress-recurse	remaining **4000** ms and **5** attempts
```